libcudacxx/test/libcudacxx/cuda/utilities/tuple/tile_only_types.pass.cpp

```
@@ -0,0 +1,88 @@
+    assert(cuda::std::get<0>(rhs) == 1337);
+  }
+}
+
+__tile_global__ void test_kernel()
+{
+  test();
+}
+
+int main(int arg, char** argv)
+{
+  NV_IF_TARGET(NV_IS_HOST, (test_kernel<<<1, 1>>>();))
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If NV_IF_TARGET is a macro then please configure it.

(unknownMacro)